app/javascript/packages/document-capture/components/document-capture.jsx

```diff
@@ -120,6 +120,7 @@ function DocumentCapture({ isAsyncForm = false, onStepChange }) {
             submissionError instanceof UploadFormEntriesError
               ? withProps({
                   remainingAttempts: submissionError.remainingAttempts,
+                  isFailedResult: submissionError.isFailedResult,
```

Comment:
just want to clarify for myself - is isFailedResult for any id upload failure of for the kind that would not trigger an option for in person proofing (ie barcode can't be read)?

Reply:
The way it's being treated here is as a "hard failure", meaning that it's a document which was recognized as unacceptable (fraudulent, etc.). This is one of the results which can be returned from our document authentication vendors. It's also distinct from other non-successful results like "unknown" or "attention". The ticket is making it so that we can continue showing the troubleshooting option for certain unsuccessful results (unknown, attention), but not for failures.

https://github.com/18F/identity-idp/blob/17d7e30e24c4cacb7801fe92e313c2c82e09338a/app/services/doc_auth/acuant/result_codes.rb#L8-L19